README.md

```diff
@@ -146,6 +146,16 @@ all languages.
 
 ## Release Notes
 
+### Release 2.0.5 (November 11, 2021)
+* Upgraded multiple dependencies [PR #148](https://github.com/awslabs/amazon-kinesis-client-python/pull/148)
+  * Amazon Kinesis Client Library 2.3.8
+  * AWS SDK 2.17.52
+* Added dependencies 
+  * AWS SDK json-utils
+  * third-party-jackson 2.17.52
```

Comment:
Minor: looks like we are adding `third-party-jackson-core` and `third-party-jackson-dataformat-cbor`, not sure if they make up `third-party-jackson` but might be nice to list the specific dependencies we are adding